src/main/java/com/nha/abdm/wrapper/hrp/mongo/tables/Patients.java

```diff
@@ -1,6 +1,6 @@
 package com.nha.abdm.wrapper.hrp.mongo.tables;
```

Comment:
The name of the class can be just `Patient`, no plurals on the class names

Reply:
The class names has been modified accordingly.